src/inpar/4C_inpar_solver.cpp

```diff
@@ -57,17 +57,8 @@ namespace Inpar::SOLVER
                 .default_value = Core::LinearSolver::PreconditionerType::ilu}),
 
         // Ifpack options
-        parameter<int>("IFPACKOVERLAP",
-            {.description = "The amount of overlap used for the ifpack \"ilu\" preconditioner.",
-                .default_value = 0}),
-
-        parameter<int>("IFPACKGFILL",
-            {.description = "The amount of fill allowed for an internal \"ilu\" preconditioner.",
-                .default_value = 0}),
-
-
-        deprecated_selection<std::string>("IFPACKCOMBINE", {"Add", "Insert", "Zero"},
-            {.description = "Combine mode for Ifpack Additive Schwarz", .default_value = "Add"}),
+        Core::IO::InputSpecBuilders::parameter<std::optional<std::filesystem::path>>(
+            "IFPACK_XML_FILE", {.description = "xml file defining any Ifpack preconditioner"}),
```

Comment:
Since these documentation strings will be used elsewhere as input to the 4C documentation, let's start to be more expressive here:
```suggestion
        Core::IO::InputSpecBuilders::parameter<std::optional<std::filesystem::path>>(
            "IFPACK_XML_FILE", {.description = "This parameter describes the absolute or relative path to an xml file containing the configuration of a Trilinos/Ifpack preconditioner. The content of this xml file needs to follow Ifpack guidelines. Consult the Trilinos/Ifpack documentation and user guide for more information on valid Ifpack parameters.."}),
```